README.md

```diff
@@ -298,6 +298,7 @@ Network-operator supports [Pod Security Policies](https://kubernetes.io/docs/con
 * Operating Systems: Ubuntu 20.04 LTS
 
 >__NOTE__: As more driver containers are built the operator will be able to support additional platforms.
+>__NOTE__: ConnectX-6 Lx is not supported.
```

Comment:
we should have a new section under System Requirements a "Supported NICs" section.
where we list the NICs we are testing against.

need to update this here and in official docs

Reply:
done